RELEASE/scripts/autoscend/auto_zone.ash

```diff
@@ -330,22 +330,10 @@ generic_t zone_needItem(location loc)
 		}
 		break;
 	case $location[The Haunted Pantry]:
```

Comment:
These empty case statements could be removed but again it's really just cosmetic so it's your choice if you want to remove them or not.